eth/blockwatch/block_watcher.go

```diff
@@ -121,7 +121,7 @@ func (w *Watcher) Watch(ctx context.Context) error {
 			return nil
 		case <-ticker.C:
 			if err := w.pollNextBlock(); err != nil {
-				glog.Errorf("blockwatch.Watcher error encountered: %v", err)
+				glog.V(6).Infof("blockwatch.Watcher error encountered - trying again on next polling interval err=%v", err)
```

Comment:
While the error here is probably recoverable, I think it makes sense to still use `Errorf()` since any error, even if recoverable, is not a part of normal program execution.

Reply:
But `Errorf` doesn't allow for verbosity level

Reply:
Fixed in 32ee65fc